modules/mco-update-boot-images.adoc

```diff
@@ -17,9 +17,9 @@ This process could cause the following issues:
 * Certificate expiration issues
 * Version skew issues
 
-To avoid these issues, you can configure your cluster to update the boot image whenever you update your cluster. By modifying the `MachineConfiguration` object, you can enable this feature. Currently, the ability to update the boot image is available for only Google Cloud Platform (GCP) clusters and is not supported for {cluster-capi-operator} managed clusters.
+To avoid these issues, you can configure your cluster to update the boot image whenever you update your cluster. By modifying the `MachineConfiguration` object, you can enable this feature. Currently, the ability to update the boot image is available for only Google Cloud Platform (GCP) clusters as a Technology Preview feature and is not supported for {cluster-capi-operator} managed clusters.
```

Comment:
Out of scope but since we're looking at attributes, maybe you can replace this with an attribute while you're at it:
```suggestion
To avoid these issues, you can configure your cluster to update the boot image whenever you update your cluster. By modifying the `MachineConfiguration` object, you can enable this feature. Currently, the ability to update the boot image is available for only {gcp-first} clusters as a Technology Preview feature and is not supported for {cluster-capi-operator} managed clusters.
```